modal/_mount_utils.py

```diff
@@ -1,19 +1,20 @@
 # Copyright Modal Labs 2022
 import os
 import posixpath
-from pathlib import PurePath
+from pathlib import PurePath, PurePosixPath
 from typing import TYPE_CHECKING, Dict, List, Tuple, Union
 
 from .exception import InvalidError
+from .volume import _Volume
```

Comment:
Is moving this import statement here OK? Why was it conditionally imported in the first place?

I need this to do `isinstance(volume, _Volume)` in `validate_volumes`, since the same S3 mount object can be used in more than one location, but Volume cannot.